client/src/index.tsx

```diff
@@ -3,7 +3,7 @@ import ReactDOM from "react-dom";
 
 // CSS Reset
 // import '@csstools/normalize.css';
```

Comment:
Does your styling break if you uncomment this line? This was commented out as part of the migration over to using a node server and webpack and I forgot to comment it back in.